zebROS_ws/src/talon_swerve_drive_controller/src/swerve_drive_controller.cpp

```diff
@@ -1032,7 +1071,7 @@ double f_a_;
 double f_v_;
 double f_s_v_;
 double f_s_s_;
-
+double stopping_ff_;
```

Comment:
Thinking about this one some more - do we want to only apply this if using coast mode?  I'd think in brake mode we really want to stop, and I thought having a ff term would prevent brake mode from happening since the motor input is never 0?